<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usuarios</class>
 <widget class="QMainWindow" name="usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingrese un nuevo Usuario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Ingrese Datos del Nuevo Usuario de Sistema</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblContrasena">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Contraseña                :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRepiteContrasena">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Repita Contraseña    :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre                      :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtContrasenna">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblFono">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fono                           :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtContrasena">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoMaterno">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoPaterno">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblApellidoPaterno">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apellido Paterno       :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblApellidoMaterno">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apellido Materno      :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtFono">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblIdUsuario">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>21</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>170</y>
      <width>21</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>21</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>21</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>21</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
